configs/base.yml

```diff
@@ -0,0 +1,39 @@
+environment: &env
```

Comment:
Seems to work just fine. To repeat http://pay.dev.mozaws.net/ is working just fine.